### ISSUE TITLE
[Integration with Map Scanner] Set location anytime + not locked to pokestops + integration with the python pokemon map scanner

### ISSUE THREAD

**Issue by laernumoretz** [Novalys/PokemonGo-Bot-SimpleGUI#77]
Posting this as a new issue, I know Novalys already agreed to add the the "set location" to be visable at all times in, but this will give this bot a ton more kick off.

"
I disagree on the comment about this not being recommended.
I fully recommend it.
In fact what is recommended, not only that set location be available at all times, but you being able to set your location and it "NOT" lock the location to the nearest pokemon stop. (Make this toggleable perhaps)

This way, and this is huge imo, you can use Pokevision to locate pokemon you want, then simply click in the map GUI on "this" bot for the bot to catch the pokemon there, with a max speed of 50km to get there from where its currently at. Therefor no ban risk.

Side note: I personally don't use poke vision, its FAR more beneficial to setup your own python pokemon scanner, which is also on github and what I use personally. I said pokevision to use as an example.

Wouldn't it be freaking crazy if the Python pokemon scanner AND this bot were combined and you could filter in specific pokemon you want it to look and chase down?
Yeah, totally doable IMO!!!

This would also make this bot far more undetectable, since thats what players would be doing. So you're super fans would be safe! (Already are for just doing this much really)

"

In short this would make this bot capable of a click search capture sequence.
Here is the python pokemap project. 
https://github.com/AHAAAAAAA/PokemonGo-Map

It already has full filters for pokemon, stops, gyms etc. So really if set location is set to view always > not locked to pokestops, all thats needed is a pointer for the bot to go where you click in the python map if its integrated into this bot, or, we can simply click the map as is without that programs integration too, integrated is just easier but not required.

**Comment by Novalys** [comment]
Hey, thanks for the suggestion. This would require a lot of work to integrate as this runs under Python code so people wouldn't be able to just compile and run but it's indeed a good idea that might be worth of taking a look after we complete the more urgent stuff regarding the bot.

**Comment by yupdon** [comment]
Would it be a separate issue if we did a location anytime change, along with a minimum wait time to avoid soft bans? - I'm always having trouble after switching locations, and how long to wait for.
